rocky/reports/templates/partials/report_header.html

```diff
@@ -6,6 +6,7 @@
 <section class="centered">
     <div class="centered">
         <img width="320"
+             style="width: 320px"
```

Comment:
debugging left over, or actually needed?

Reply:
No it is actually needed, because Weasyprint ignores the presentational hint of the width attribute. It has an option to not do that, but that didn't seem to work and I figured it isn't worth spending a lot of time on.

I've added a comment that explains it.